src/Raven.Server/Documents/AbstractBackgroundWorkStorage.cs

```diff
@@ -61,11 +61,14 @@ private void ThrowWrongDateFormat(Slice lowerId, string expirationDate)
         throw new InvalidOperationException(
             $"The due date format for document '{lowerId}' is not valid: '{expirationDate}'. Use the following format: {Database.Time.GetUtcNow():O}");
     }
-    
-    
-
 
     public Dictionary<Slice, List<(Slice LowerId, string Id)>> GetDocuments(BackgroundWorkParameters options, out Stopwatch duration, CancellationToken cancellationToken)
+    {
+        var totalCount = 0;
+        return GetDocuments(options, ref totalCount, out duration, cancellationToken);
+    }
+
+    public Dictionary<Slice, List<(Slice LowerId, string Id)>> GetDocuments(BackgroundWorkParameters options, ref int totalCount, out Stopwatch duration, CancellationToken cancellationToken)
```

Comment:
private and rename to `GetDocumentsInternal`

Reply:
done